src/components/links/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { INavLinks } from '../navBar/helper';
+import { LinksWrapper, StyledIconWrapper, StyledLink } from './styled';
+
+export interface LocationType {
+  pathname: string;
+}
+const Links: React.FC<{ links: Array<INavLinks>; location: LocationType }> = ({
+  links,
+  location,
+}) => {
+  return (
+    <>
+      {links.map((link) => (
+        <div key={link.name}>
```

Comment:
I said that you need to move NavLink component into the separate component not a list. In this case you don't need to pass `link array` and `location`.
P.S. in general passing location through the props looks strange

Reply:
fix: NavLink as a separate component